html/partials/description_selector.html.haml

```diff
@@ -8,7 +8,7 @@
         %h4.modal-title#description-selector-label
           Select a description from the list below to
           %span.description-selector-action
-            show
+            show its details
```

Comment:
Could you add a dot at the end so that it's exactly the same with the show view?